docs/architecture/adr-0001-app-setup-and-goals.md

```diff
@@ -0,0 +1,168 @@
+# Context
+
+We need to create an application without a dependency on Raster Foundry that has
+the following capabilities:
+
+- run models on imagery
+- kick off model runs when it learns about new imagery
+- run models without knowing anything about the content of those models
+- summarize past model results
+- allow users to respond to model run events
+
+Additionally, we'd like the application to meet several design goals that aren't
+core to the pursuit of those features:
+
+- we should deploy continuously
+- infrastructure should be kept as light as possible
+
+These requirements are located roughly a third of the way from "core competency
```

Comment:
Exactly what this sentence is connected to, or intends to expand upon, is a bit unclear to me. Is this capturing the spectrum for the previous two bullets, or all of the bullets? Also, it is unclear what "a third" represents here.